packages/p2p/src/components/modal-manager/modals/error-modal/error-modal.tsx

```diff
@@ -33,9 +33,7 @@ const ErrorModal = ({
             width={is_mobile ? '90rem' : '40rem'}
         >
             <Modal.Body className='error-modal__body'>
-                <Text size='xs'>
-                    <Localize i18n_default_text={error_message} />
-                </Text>
+                <Text size='xs'>{error_message}</Text>
```

Comment:
for the other places where we show the error modal, are we passing Localize for each one through the code base?

Reply:
yes we are passing the localized message

Reply:
yeah.. wont it affect the other error messages? also reactNode accepts undefined as well.. so might need to check if error_message is there.. 
also reactnode already includes string.. so string | reactnode is not needed i guess

Reply:
yeah, wherever we are calling showModal for errorModal, are we passing the strings with Localize as well, cause ti might affect it

Reply:
We are already passing the localized message for all other error messages.

Reply:
ah alright great 👍